src/assets/sass/v2/_typography.scss

```diff
@@ -0,0 +1,59 @@
+body {
+  font-family: "nta", Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  font-size: 19px;
+  line-height: 1.31579;
+}
+
+* {
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+
+h1 a:link {
+  font-weight: normal;
+  text-decoration: none;
+}
+
+h1 a:link, h1 a:visited {
+  font-weight: normal;
+  text-decoration: none;
+}
+
+.sub-head {
+  font-size: 30px;
+  color: #6f777b;
+  margin: 0;
+  padding: 0;
+}
+
+// ========== ========== ========== ========== ==========
```

Comment:
From here to the end of the file is also present in the _styles.scss file. Is this correct?

Also I think we should probably avoid styling by id.